pkg/types/none/validation/platform.go

```diff
@@ -0,0 +1,17 @@
+package validation
+
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/openshift/installer/pkg/types/common"
+)
+
+// ValidateFencingCredentials checks that the provided fencing credentials are valid.
+func ValidateFencingCredentials(fencingCredentials []*common.FencingCredential, fldPath *field.Path) (errors field.ErrorList) {
```

Comment:
I think the validation is failing to take the DualReplica topology into account. Shouldn't there be a validation to check
* if the number of control plane replicas == 2, then there needs to be exactly two fencing credentials.
* if the number of control plane replicas != 2, then there cannot be any fencing credentials defined

Reply:
Good catch !
updated.